code/obj/submachine/robotics.dm

```diff
@@ -125,6 +161,70 @@
 		else
 			..()
 
+//Returns the turf facing the fab for cardinal directions (which should also be the user's turf), but for diagonals it returns a neighbouring turf depending on where you click
+//Just in case you're attacking a corner diagonally.
```

Comment:
```suggestion
/// Returns the turf facing the fab for cardinal directions (which should also be the user's turf), but for diagonals it returns a neighbouring turf depending on where you click
/// Just in case you're attacking a corner diagonally.
```